interface/billing/sl_eob_search.php

```diff
@@ -1071,7 +1147,19 @@ function npopup(pid) {
 
 if (!$INTEGRATED_AR) SLClose();
 ?>
-
+<tr bgcolor='<?php echo $bgcolor ?>'>
+<td>Totals</td>
```

Comment:
https://github.com/juggernautsei/openemr/blob/7e6ca38d3bc71265c0b85e798a2f54a7b4a32ff6/interface/billing/sl_eob_search.php

I have made the addition the file.

Sherwin
757-328-2736
Open Med Practice

“For those who believe, no proof is necessary. For those who don't believe,
no proof is possible.” ― Stuart Chase
<http://www.goodreads.com/author/show/1051789.Stuart_Chase>

On Tue, Feb 23, 2016 at 3:54 AM, Brady Miller <notifications@github.com>
wrote:

> In interface/billing/sl_eob_search.php
> <https://github.com/openemr/openemr/pull/98#discussion_r53751327>:
>
> > @@ -1071,7 +1147,19 @@ function npopup(pid) {
> >
> >  if (!$INTEGRATED_AR) SLClose();
> >  ?>
> > -
> > +<tr bgcolor='<?php echo $bgcolor ?>'>
> > +<td>Totals</td>
>
> Hi Sherwin,
> But I'm not asking to translate the variables, just the 'Totals' string.
> -brady
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openemr/openemr/pull/98/files#r53751327>.
>